apps/yapms/src/lib/components/sidebar/sections/savedmaps/SaveMap.svelte

```diff
@@ -37,13 +38,13 @@
 		class="input input-bordered input-sm flex-grow overflow-hidden join-item"
 		bind:value={newMapName}
 		placeholder="Map Name"
-		disabled={disabled || submitting}
+		disabled={disabled || submitting || $ImportedSVGStore.loaded}
 	/>
 	<div class="tooltip" data-tip="Create">
 		<button
 			class="btn btn-sm btn-primary join-item"
 			on:click={createMap}
-			disabled={disabled || submitting}
+			disabled={disabled || submitting || $ImportedSVGStore.loaded}
```

Comment:
this condition is bad- $ImportedSVGStore.loaded will still have content if you import a map and then leave that page and navigate to another.

Reply:
do we lock it based on path??

Reply:
I think the state should be changed to false if a map isn't loaded.

Reply:
im cool w/ that, if you wanna have this PR manage that and then use this condition that is good

Reply:
Eh, your idea looks more clear in the code I think.